app/src/constants/paths.js

```diff
@@ -2,3 +2,4 @@ import {normalize} from 'path'
 
 export const lib = normalize(`${__dirname}/..`)
 export const root = normalize(`${lib}/../..`)
+export const chat = 'chat'
```

Comment:
Its  a wrong place for this. Its a part of a path of a  url, where the others are full paths on the file system.